### PR TITLE
Add iframe message handling (integration creation) [#168661241]

### PR DESCRIPTION
**Pull request by vitaliypodlypnyi**

### CODE REVIEW

File: CredentialPostTest/CredentialPostTest/wwwroot/js/site.js [hunk 1]
```diff
@@ -2,3 +2,26 @@
...
+    if (element.addEventListener) {
+        element.addEventListener(eventName, eventHandler, false);
+    } else if (element.attachEvent) {
+        element.attachEvent('on' + eventName, eventHandler);
+    }
+}
+
+//here goes the code which should be run on iframe parent when integration is created 
+function processResult(result) {
+    let systems = result.split('|');
```
Comment (KaterynaZorina) [inline CredentialPostTest/CredentialPostTest/wwwroot/js/site.js:15]: We subscribed to all `message` events here, so we can receive not only messages from iFrame. At least we need to add check that `systems` contains 2 elements and process this case further.

Comment (vitaliypodlypnyi) [reply CredentialPostTest/CredentialPostTest/wwwroot/js/site.js:15]: fixed

**Comment by dennisahlin** [comment]
Merged into `master`? :S

**Comment by KaterynaZorina** [comment]
@dennisahlin It's an example project. We don't have `develop` branch for it, only master.

**Comment by dennisahlin** [comment]
Ah, sorry, didn't look at which repo it was -.-

**Pull request merged**